src/backlinks/handler.js

```diff
@@ -143,6 +133,24 @@ export default async function auditBrokenBacklinks(message, context) {
       auditContext,
       auditResult,
     };
+
+    try {
+      const baseUrl = site.getBaseURL();
+      const sitemaps = await findSitemap(baseUrl);
```

Comment:
The method name is misleading, as it seems `findSitemap` actually returns the page URLs listed in the sitemaps?